### DISCUSSION TITLE
"Submarine Tiers" doesn't fix "Deep-Diver" just moves the problem somewhere else.

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#9816] [Unstable feedback]
> - Removed the Deep Diver class: the way we see it, Deep Divers didn't have a clear enough role in the game, especially considering that hull upgrades served pretty much the same purpose. In practice, the only clear benefit of a Deep Diver was being able to get through the very last levels of the campaign, and having to switch to one just for that purpose wasn't fun. Now any submarine with full hull upgrades can get all the way to the end of the campaign.
> - Added submarine tiers. Higher-tier submarines can be upgraded further than lower-tier submarines.

The "Tiers" system means you're still locked out of progression and must change submarines, just now non-deepdiver submarines has been replaced with this nebulous concept of designating a submarine as "low tier".

**Comment by ubertpendragon** [comment]
Since you're supposed to start with a limited amount of money, and since bigger subs cost more money, swapping mid-campaign was always intended

There are 3 tiers but I assume you'll only swap submarines once; most players will also make their submarine T3 always for obvious reasons but the price can wildly vary.

Submarine cost dictates tier:
Under 10000mk is T1
Over 20000mk is T3

For Tiers and depth, at max hull upgrades: 
-T1 crush depth starts around the 3rd biome
-T2 submarine *can* reach the end of the game without entering crush depth.

There is good reason to skip T2 entirely in this tiered system

**Comment by Ek-Videogames** [comment]
> There are 3 tiers but I assume you'll only swap submarines once; most players will also make their submarine T3 always for obvious reasons but the price can wildly vary.

The existence of Tiers 1 and 2 seems like just more reasons for people to never bother buying subs like the Azimuth/Typhon2/Big-Rig.

Basically what I mean is, this update doesn't really accomplish the stated goal of **making *"Non-deep-diver"*-tiered submarines viable**, 
just re-conceptualizes what a *"deep-diver"* is & removes the crush depth benefit.

**Comment by FalcoGer** [comment]
I find the concept not very nice to begin with. You not only upgrade your submarine, but you also get attached to it. Then suddenly you are locked out of progression because of some arbitrary money value assigned to it. This will disincentivise people from upgrading their early submarine as it will waste money after they are forced ditch it later anyway.

When I start a campaign I see the "pick a submarine" list as something like "What ship do you want to play with for the entirety of the game." Changing subs midway through is just a nightmare. All your upgrades are gone, loads of money wasted. Then you have to do the logicistal nightmare of putting all the things where they belong in the order they belong. Any custom wire jobs have to be redone, it's just no fun at all. If you wanted to play with some sub then pick it to begin with.

An easy fix would be to simply allow you to upgrade your tier on the submarine you already own.

**Reply by gerakis** [reply]
I agree with this. Just get rid of the tier system entirely. If they want to use a barsuk the entire campaign I dont see how thats a problem.

**Comment by TimFragmagnet** [comment]
Not only does the tier system not solve the deep diver issue, it also discourages players from buying specific submarines.

With the new submarine tiers, you are restricted in how far you can upgrade a submarine based on its tier, this means that the player is naturally guided towards higher tier subs to get the higher upgrades, that's honestly fine by me.

The problem is that progressing from T1 to T2 to T3 is a waste of time and money. It is far more efficient to just go from T1 to T3.
The reason is that the cost of T3 subs is relatively low compared to the cost of T2 subs, just do around 5-10 missions and you have enough money for a T3, so why buy a T2.

The solution is fairly simple, increase the cost of T2 and T3 submarines, and then allow the player to sell/trade in their submarine for a large fraction of its value if not full value.

T3 subs being very very expensive discourages the player from just grinding straight to them with a T1 submarine, instead, the ability to sell/trade in the submarines means that the player is encouraged to buy a T2 submarine and proceed with that. Then once the time comes to buy a T3 submarine, its high price is reduced by the cost of the T2 submarine, effectively turning the T2 submarine into an investment.
If you buy an expensive T2 submarine, the cost of your T3 submarine is that much lower, while at the same time you're still able to work on further submarine upgrades because you aren't locked by a T1.

This would at least alleviate the bad feeling of the tier system as it currently stands.

**Comment by MateusJK15** [comment]
> You not only upgrade your submarine, but you also get attached to it

Specially the design, like machinery layout, turrets positions, turrets blind spots, flood containment capability, logistics, lighting.

**Buying a more expensive/higher tier submarine DOES NOT means buying a better submarine**

**Comment by NichikaZ** [comment]
Humpback all the way

**Comment by MapleWheels** [comment]
I agree, I also grow attached to the submarine, especially when I customize the crap out of it using @Ek-Videogames utilities and whatnot :).

What they need to embrace is the idea that different subs are built with different ideas in mind. Make it so that the maximum upgrades for given items are limited by class only. This means that the class you designate determines how you intend the sub to be used.

When we talk about costs, we need to keep in mind that--literally--everything in this game is customizable in the editor or in XML, so most of this locking down is futile anyways, we can just circumvent it. The system needs to make creators want to embrace it, not feel like we're fighting with it.

 I feel like what the devs are trying to accomplish is to make the "hidden" elements of a submarine exist (hull integrity, etc) and give a sense of progression. The way you need to do this is by offering something **late-game** that people don't want **early-game**. An example of this is something like *operating costs*, which the game doesn't implement very well (an example of this implemented is Mechtrauma). 

Assuming we could overhaul operational complexity and costs, make Tier 3 submarines much stronger while costing more to use and upkeep. Something that you *want* but actually can't *afford*. 

If this sounds like the current system, it's not. Why? Well;
- Multiplayer Campaigns bypass the cost limits.
- Upkeep costs and complexity do not exist, current sub-designers do not make their sub around the Tier system.
- **There is no real tie-in between operational complexity (using controls, maintenance, features, etc.) and character progression (Talents, Skill, etc.) besides "make the game easier."**

The last point to me is the important one. Which sub you use should be tied to your intended playstyle and crew. We already do this with the Character Talents system, why are we not doing this here? For example, the Captain rework in the new talents honestly sucks. It's very powerful and at the same time as Captain, I feel like an NPC. There's nothing *new* that I really end up doing as a result of my talents. I would live it if I got new talents that changed my gameplay mid-game and then later, got a new sub that worked/synergizes with my new talents.

**Comment by Ek-Videogames** [comment]
This is more of a "content" problem than a balance one: 
(Thought it could be argued balance decisions is how we got into this mess in the first place) 
-but with submarine options being **further restricted** with more changes to upgrade restrictions & now some submarines being locked from purchase, there is simply not enough variety of base-game vanilla submarines to properly cover the full gamut of roles, tiers, and of course "crew playstyle"
